deployment/permissions/service.userapprole.ini

```diff
@@ -0,0 +1,45 @@
+[action_permission_items]
```

Comment:
General comments
1. why do we need batch & admin console impersonate allowed here?
2. Why don't we allow EP impersonated calls? do we need this?

Reply:
1. No need, removed
2. EP can already do session -> impersonate
so they make api requests via the impersonated ks